gdnative-core/src/nativescript/init.rs

```diff
@@ -249,10 +251,19 @@ impl<C: NativeClass> ClassBuilder<C> {
     #[inline]
     pub fn add_method_advanced(&self, method: ScriptMethod) {
         let method_name = CString::new(method.name).unwrap();
-        let attr = sys::godot_method_attributes {
-            rpc_type: sys::godot_method_rpc_mode_GODOT_METHOD_RPC_MODE_DISABLED,
+
+        let rpc = match method.attributes.rpc_mode {
+            RpcMode::Master => sys::godot_method_rpc_mode_GODOT_METHOD_RPC_MODE_MASTER,
+            RpcMode::Remote => sys::godot_method_rpc_mode_GODOT_METHOD_RPC_MODE_REMOTE,
+            RpcMode::Puppet => sys::godot_method_rpc_mode_GODOT_METHOD_RPC_MODE_SLAVE,
+            RpcMode::RemoteSync => sys::godot_method_rpc_mode_GODOT_METHOD_RPC_MODE_SYNC,
```

Comment:
Shouldn't the canonical constants be used? Are they not declared in the header?

Reply:
Yep they are there in the header, mistake on my part